Update/kage_010.txt

```diff
@@ -1992,7 +1992,7 @@ void main()
 //rそう、これは不幸な事故。…だから、二人にとっては家族の死よりも、そのせいで今までの生活が失われることのほうが恐ろしくて、…ワタシの意のままに動いてくれた。
 	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
 	OutputLine(NULL, "そう、これは不幸な事故。…だから、二人にとっては家族の死よりも、そのせいで今までの生活が失われることのほうが恐ろしくて、…ワタシの意のままに動いてくれた。",
-			NULL, "Yes, this is an unfortunate accident. ...So for the two of them, it's more than just losing a family member. It's a reminder of how terrifyingly close death lurks.", Line_Normal);
+			NULL, "Yes, this is an unfortunate accident. ...So for the two of them, they're acting according to my will not because a family member died... but because they're afraid they'll lose their current lives as a result.", Line_Normal);
```

Comment:
```suggestion
			NULL, "Yes, this is an unfortunate accident. ...So for the two of them, they're acting according to my will not because a family member died... but because they're afraid they'll lose their current way of life as a result.", Line_Normal);
```